tests/test_data.py

```diff
@@ -6,6 +6,22 @@
 from pgactivity.data import Data
 
 
+def wait_for_data(
+    fct, timeout: int = 2, desc_recursionError: str = "timeout in wait_for_data"
+):
+    count = int(timeout / 0.1)
+    for _ in range(count):
+        time.sleep(0.1)
+        data = fct()
+
+        if not data:
+            continue
+        break
+    else:
+        raise RecursionError(desc_recursionError)
```

Comment:
`AssertionError` would be more appropriate.

Reply:
Fixed in 6e080de820bfff697476e45d0bcb69898cc967a6